homework.10/homework.10/randomEmojiViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ViewController.swift
+//  homework.10
+//
+//  Created by Stanislau on 01.04.2022.
+//
+
+import UIKit
+
+class randomEmojiViewController: UIViewController {
+
+    @IBOutlet weak var showButton: UIButton!
+    @IBOutlet weak var nextViewButton: UIButton!
```

Comment:
It's better to create a container and create a position calculation according to its parameters instead of adding these buttons. Your code can break if you change the buttons layout